### PR TITLE
Verbosity control for export

### PR DESCRIPTION
**Pull request by mikekgfb**
Summary: Verbosity control for export

Reviewed By: kimishpatel

Differential Revision: D55374748

### CODE REVIEW

**Review by kirklandsign — APPROVED** [review]

**Review by manuelcandales — APPROVED** [review]

File: examples/models/llama2/export_llama_lib.py [hunk 1]
```diff
@@ -57,6 +58,15 @@ def get_resource_path(resource_name) -> str:
     return pkg_resources.resource_filename(pkg_name, resource_name)
 
 
+def set_verbosity(val):
+    global verbosity_setting
+    verbosity_setting = val
+
+
+def verbose_export():
+    return verbosity_setting
```
Comment (dbort) [inline examples/models/llama2/export_llama_lib.py:67]: Would be good to clarify the type here. And along with it, `verbosity_setting: bool = False` above.

```suggestion
def set_verbosity(val: bool):
    global verbosity_setting
    verbosity_setting = val


def verbose_export() -> bool:
    return verbosity_setting
```

File: examples/models/llama2/export_llama_lib.py [hunk 2]
```diff
@@ -46,6 +46,7 @@
 logging.basicConfig(level=logging.INFO, format=FORMAT)
 
 pkg_name = __name__
+verbosity_setting = None
```
Comment (dbort) [inline examples/models/llama2/export_llama_lib.py:49]: Might be good to make this clearly internal, though I don't know if anyone imports this file.

```suggestion
_verbosity_setting = None
```

Comment (dbort) [inline examples/models/llama2/export_llama_lib.py:49]: Is there any way for a user to override this, or is it effectively hard-disabled for now?

File: examples/models/llama2/export_llama_lib.py [hunk 3]
```diff
@@ -240,7 +250,8 @@ def quantize(
         from torchao.quantization.quant_api import Int8DynActInt4WeightQuantizer
 
         model = Int8DynActInt4WeightQuantizer(precision=torch_dtype).quantize(model)
-        print("quantized model:", model)
+        if verbose_export():
```
Comment (dbort) [inline examples/models/llama2/export_llama_lib.py:253]: If the idea is to guard all prints with this, consider adding a custom print function like `log_export()` that does it internally to simplify the call sites.

**Review by dbort — APPROVED** [review]
Ok as-is, but left some suggestions for improving clarity.

**Pull request closed without merging**